jane/doc/extensions/stack/reference.md

```diff
@@ -179,10 +180,25 @@ value `outer`, while being local, was definitely not local to the region of
 compiler know that it is safe to still refer to `outer` from within the closure
 `g`? See "Closures" below for more details)
 
+This situation also arises with local parameters. For example:
+
+```ocaml
+let f (local_ x) =
+  let local_ y = 3 :: x in
+  ??
+```
+
+Both `x` and `y` are local and cannot, in general, escape a region. However,
+filling `??` in with `x` (but not `y`) is allowed. This is because we know that
+the value pointed to by `x` must have existed before `f` started and thus is
+guaranteed to be allocated in a pre-existing stack frame (or on the heap).
```

Comment:
thanks - this is an important addition.

reminder to myself: I think the interaction between allocation and locality is subtle. I will try to see if can be made clearer.